### PR TITLE
feat(os) : support Alma Linux

### PR DESCRIPTION
**Pull request by kazuminn**
# What did you implement:
support Alma Linux version 8.4.
https://almalinux.org/

on oval.
```
$vuls report #fast mode
[Jun 25 22:53:42]  INFO [localhost] vuls-v0.15.11-build-20210625_211447_8e6351a
[Jun 25 22:53:42]  INFO [localhost] Validating config...
[Jun 25 22:53:42]  INFO [localhost] cveDict.type=sqlite3, cveDict.url=, cveDict.SQLite3Path=/Users/a/vuls/cve.sqlite3
[Jun 25 22:53:42]  INFO [localhost] ovalDict.type=sqlite3, ovalDict.url=, ovalDict.SQLite3Path=/Users/a/vuls/oval.sqlite3
[Jun 25 22:53:42]  INFO [localhost] gost.type=sqlite3, gost.url=, gost.SQLite3Path=/Users/a/vuls/gost.sqlite3
[Jun 25 22:53:42]  INFO [localhost] exploit.type=sqlite3, exploit.url=, exploit.SQLite3Path=/Users/a/vuls/go-exploitdb.sqlite3
[Jun 25 22:53:42]  INFO [localhost] metasploit.type=sqlite3, metasploit.url=, metasploit.SQLite3Path=/Users/a/vuls/go-msfdb.sqlite3
[Jun 25 22:53:42]  INFO [localhost] Loaded: /Users/a/vuls/results/2021-06-25T22:53:11+09:00
[Jun 25 22:53:42]  INFO [localhost] OVAL redhat 8.4 found. defs: 598
[Jun 25 22:53:42]  INFO [localhost] OVAL redhat 8.4 is fresh. lastModified: 2021-06-25T18:54:00+09:00
[Jun 25 22:53:43]  INFO [localhost] [Reboot Required] alma: 2 CVEs are detected with OVAL
[Jun 25 22:53:43]  INFO [localhost] [Reboot Required] alma: 0 unfixed CVEs are detected with gost
[Jun 25 22:53:43]  INFO [localhost] [Reboot Required] alma: 0 CVEs are detected with CPE
[Jun 25 22:53:43]  INFO [localhost] [Reboot Required] alma: 0 PoC are detected
[Jun 25 22:53:43]  INFO [localhost] [Reboot Required] alma: 0 exploits are detected
[Reboot Required] alma (redhat8.4)
==================================
Total: 2 (Critical:0 High:2 Medium:0 Low:0 ?:0)
2/2 Fixed, 0 poc, 0 exploits, en: 0, ja: 0 alerts
368 installed

+---------------+------+--------+-----+------+---------+------------------------------------------------+
|    CVE-ID     | CVSS | ATTACK | POC | CERT |  FIXED  |                      NVD                       |
+---------------+------+--------+-----+------+---------+------------------------------------------------+
| CVE-2021-3501 |  8.9 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3501 |
| CVE-2021-3543 |  8.9 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3543 |
+---------------+------+--------+-----+------+---------+------------------------------------------------+


```

```
$vuls report # fast-root mode
[Jun 26 22:57:05]  INFO [localhost] exploit.type=sqlite3, exploit.url=, exploit.SQLite3Path=/Users/a/vuls/go-exploitdb.sqlite3
[Jun 26 22:57:05]  INFO [localhost] metasploit.type=sqlite3, metasploit.url=, metasploit.SQLite3Path=/Users/a/vuls/go-msfdb.sqlite3
[Jun 26 22:57:05]  INFO [localhost] Loaded: /Users/a/vuls/results/2021-06-26T22:56:21+09:00
[Jun 26 22:57:05]  INFO [localhost] OVAL redhat 8.4 found. defs: 598
[Jun 26 22:57:05]  INFO [localhost] OVAL redhat 8.4 is fresh. lastModified: 2021-06-25T18:54:00+09:00
[Jun 26 22:57:06]  INFO [localhost] [Reboot Required] alma: 2 CVEs are detected with OVAL
[Jun 26 22:57:06]  INFO [localhost] [Reboot Required] alma: 0 unfixed CVEs are detected with gost
[Jun 26 22:57:06]  INFO [localhost] [Reboot Required] alma: 0 CVEs are detected with CPE
[Jun 26 22:57:06]  INFO [localhost] [Reboot Required] alma: 0 PoC are detected
[Jun 26 22:57:06]  INFO [localhost] [Reboot Required] alma: 0 exploits are detected
[Reboot Required] alma (redhat8.4)
==================================
Total: 2 (Critical:0 High:2 Medium:0 Low:0 ?:0)
2/2 Fixed, 0 poc, 0 exploits, en: 0, ja: 0 alerts
370 installed

+---------------+------+--------+-----+------+---------+------------------------------------------------+
|    CVE-ID     | CVSS | ATTACK | POC | CERT |  FIXED  |                      NVD                       |
+---------------+------+--------+-----+------+---------+------------------------------------------------+
| CVE-2021-3501 |  8.9 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3501 |
| CVE-2021-3543 |  8.9 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3543 |
+---------------+------+--------+-----+------+---------+------------------------------------------------+
```

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:
You don't have to satisfy all of the following.

- [ ] Write tests
- [x] Write documentation
- [x] Check that there aren't other open pull requests for the same issue/feature
- [x] Format your source code by `make fmt`
- [x] Pass the test by `make test`
- [ ] Provide verification config / commands
- [x] Enable "Allow edits from maintainers" for this PR
- [x] Update the messages below

***Is this ready for review?:*** YES

### CODE REVIEW

File: scanner/alma.go [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+	"github.com/future-architect/vuls/logging"
+	"github.com/future-architect/vuls/models"
+)
+
+// inherit OsTypeInterface
+type alma struct {
+	redhatBase
+}
+
+// NewAmazon is constructor
```
Comment (MaineK00n) [inline scanner/alma.go:14]: Isn't this a typo? (NewAmazon → NewAlma)

**Review by MaineK00n — CHANGES_REQUESTED** [review]
Are you able to test this correctly?

Looking at this log, you are scanning for `redhat8.4`. Shouldn't this part be `alma8.4`?
```
[Reboot Required] alma (redhat8.4)
```

I would like to know more about your test environment, because it is possible that something is wrong with mine.

**Comment by kazuminn** [comment]
@MaineK00n 
> Looking at this log, you are scanning for redhat8.4. Shouldn't this part be alma8.4?

Maybe, I fixed. Becase it's "alma (alma8.4)".



> I would like to know more about your test environment, because it is possible that something is wrong with mine.

I installed the latest version of alma. And I updated the kernel with yum, but forgot to reboot.



EOL info link : https://wiki.almalinux.org/FAQ.html

Below is the latest version of the scan log.

```
$ vuls scan && vuls report
[Jul 10 12:19:51]  INFO [localhost] vuls-v0.15.11-build-20210710_121940_b0af72a
[Jul 10 12:19:51]  INFO [localhost] Start scanning
[Jul 10 12:19:51]  INFO [localhost] config: /Users/a/vuls/config.toml
[Jul 10 12:19:51]  INFO [localhost] Validating config...
[Jul 10 12:19:51]  INFO [localhost] Detecting Server/Container OS... 
[Jul 10 12:19:51]  INFO [localhost] Detecting OS of servers... 
[Jul 10 12:19:51]  INFO [localhost] (1/1) Detected: alma: alma 8.4
[Jul 10 12:19:51]  INFO [localhost] Detecting OS of containers... 
[Jul 10 12:19:51]  INFO [localhost] Checking Scan Modes... 
[Jul 10 12:19:51]  INFO [localhost] Detecting Platforms... 
[Jul 10 12:19:53]  INFO [localhost] (1/1) alma is running on other
[Jul 10 12:19:53]  INFO [alma] Scanning OS pkg in fast mode
[Jul 10 12:20:01]  INFO [alma] Scanning listen port...
[Jul 10 12:20:01]  INFO [alma] Using Port Scanner: Vuls built-in Scanner


Scan Summary
================
alma	alma8.4	371 installed, 39 updatable





To view the detail, vuls tui is useful.
To send a report, run vuls report -h.
[Jul 10 12:20:01]  INFO [localhost] vuls-v0.15.11-build-20210710_121940_b0af72a
[Jul 10 12:20:01]  INFO [localhost] Validating config...
[Jul 10 12:20:01]  INFO [localhost] cveDict.type=sqlite3, cveDict.url=, cveDict.SQLite3Path=/Users/a/vuls/cve.sqlite3
[Jul 10 12:20:01]  INFO [localhost] ovalDict.type=sqlite3, ovalDict.url=, ovalDict.SQLite3Path=/Users/a/vuls/oval.sqlite3
[Jul 10 12:20:01]  INFO [localhost] gost.type=sqlite3, gost.url=, gost.SQLite3Path=/Users/a/vuls/gost.sqlite3
[Jul 10 12:20:01]  INFO [localhost] exploit.type=sqlite3, exploit.url=, exploit.SQLite3Path=/Users/a/vuls/go-exploitdb.sqlite3
[Jul 10 12:20:01]  INFO [localhost] metasploit.type=sqlite3, metasploit.url=, metasploit.SQLite3Path=/Users/a/vuls/go-msfdb.sqlite3
[Jul 10 12:20:01]  INFO [localhost] Loaded: /Users/a/vuls/results/2021-07-10T12:19:53+09:00
[Jul 10 12:20:01]  INFO [localhost] OVAL alma 8.4 found. defs: 612
[Jul 10 12:20:01]  INFO [localhost] OVAL alma 8.4 is fresh. lastModified: 2021-07-09T21:47:11+09:00
[Jul 10 12:20:02]  INFO [localhost] alma: 20 CVEs are detected with OVAL
[Jul 10 12:20:02]  INFO [localhost] alma: 0 unfixed CVEs are detected with gost
[Jul 10 12:20:02]  INFO [localhost] alma: 0 CVEs are detected with CPE
[Jul 10 12:20:02]  INFO [localhost] alma: 0 PoC are detected
[Jul 10 12:20:02]  INFO [localhost] alma: 0 exploits are detected
alma (alma8.4)
==============
Total: 20 (Critical:0 High:13 Medium:6 Low:1 ?:0)
20/20 Fixed, 0 poc, 0 exploits, en: 0, ja: 0 alerts
371 installed

+----------------+------+--------+-----+------+---------+-------------------------------------------------+
|     CVE-ID     | CVSS | ATTACK | POC | CERT |  FIXED  |                       NVD                       |
+----------------+------+--------+-----+------+---------+-------------------------------------------------+
| CVE-2020-26541 |  8.9 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-26541 |
| CVE-2020-8616  |  8.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8616  |
| CVE-2020-8617  |  8.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8617  |
| CVE-2020-8625  |  8.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8625  |
| CVE-2021-25215 |  8.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-25215 |
| CVE-2021-33034 |  8.9 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-33034 |
| CVE-2021-3517  |  8.6 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3517  |
| CVE-2021-3518  |  8.6 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3518  |
| CVE-2021-3520  |  8.6 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3520  |
| CVE-2021-3516  |  7.8 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3516  |
| CVE-2019-6477  |  7.5 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2019-6477  |
| CVE-2020-8623  |  7.5 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8623  |
| CVE-2021-3537  |  7.5 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3537  |
| CVE-2020-8619  |  6.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8619  |
| CVE-2020-8622  |  6.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8622  |
| CVE-2020-8624  |  6.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8624  |
| CVE-2021-20271 |  6.9 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-20271 |
| CVE-2021-3541  |  6.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3541  |
| CVE-2018-5745  |  4.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2018-5745  |
| CVE-2019-6465  |  3.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2019-6465  |
+----------------+------+--------+-----+------+---------+-------------------------------------------------+

$ vuls scan && vuls report
[Jul 10 12:20:16]  INFO [localhost] vuls-v0.15.11-build-20210710_121940_b0af72a
[Jul 10 12:20:16]  INFO [localhost] Start scanning
[Jul 10 12:20:16]  INFO [localhost] config: /Users/a/vuls/config.toml
[Jul 10 12:20:16]  INFO [localhost] Validating config...
[Jul 10 12:20:16]  INFO [localhost] Detecting Server/Container OS... 
[Jul 10 12:20:16]  INFO [localhost] Detecting OS of servers... 
[Jul 10 12:20:16]  INFO [localhost] (1/1) Detected: alma: alma 8.4
[Jul 10 12:20:16]  INFO [localhost] Detecting OS of containers... 
[Jul 10 12:20:16]  INFO [localhost] Checking Scan Modes... 
[Jul 10 12:20:16]  INFO [localhost] Detecting Platforms... 
[Jul 10 12:20:17]  INFO [localhost] (1/1) alma is running on other
[Jul 10 12:20:17]  INFO [alma] Scanning OS pkg in fast-root mode
[Jul 10 12:20:39]  INFO [alma] Scanning listen port...
[Jul 10 12:20:39]  INFO [alma] Using Port Scanner: Vuls built-in Scanner


Scan Summary
================
alma	alma8.4	371 installed, 39 updatable





To view the detail, vuls tui is useful.
To send a report, run vuls report -h.
[Jul 10 12:20:39]  INFO [localhost] vuls-v0.15.11-build-20210710_121940_b0af72a
[Jul 10 12:20:39]  INFO [localhost] Validating config...
[Jul 10 12:20:39]  INFO [localhost] cveDict.type=sqlite3, cveDict.url=, cveDict.SQLite3Path=/Users/a/vuls/cve.sqlite3
[Jul 10 12:20:39]  INFO [localhost] ovalDict.type=sqlite3, ovalDict.url=, ovalDict.SQLite3Path=/Users/a/vuls/oval.sqlite3
[Jul 10 12:20:39]  INFO [localhost] gost.type=sqlite3, gost.url=, gost.SQLite3Path=/Users/a/vuls/gost.sqlite3
[Jul 10 12:20:39]  INFO [localhost] exploit.type=sqlite3, exploit.url=, exploit.SQLite3Path=/Users/a/vuls/go-exploitdb.sqlite3
[Jul 10 12:20:39]  INFO [localhost] metasploit.type=sqlite3, metasploit.url=, metasploit.SQLite3Path=/Users/a/vuls/go-msfdb.sqlite3
[Jul 10 12:20:39]  INFO [localhost] Loaded: /Users/a/vuls/results/2021-07-10T12:20:17+09:00
[Jul 10 12:20:39]  INFO [localhost] OVAL alma 8.4 found. defs: 612
[Jul 10 12:20:39]  INFO [localhost] OVAL alma 8.4 is fresh. lastModified: 2021-07-09T21:47:11+09:00
[Jul 10 12:20:40]  INFO [localhost] alma: 20 CVEs are detected with OVAL
[Jul 10 12:20:40]  INFO [localhost] alma: 0 unfixed CVEs are detected with gost
[Jul 10 12:20:40]  INFO [localhost] alma: 0 CVEs are detected with CPE
[Jul 10 12:20:40]  INFO [localhost] alma: 0 PoC are detected
[Jul 10 12:20:40]  INFO [localhost] alma: 0 exploits are detected
alma (alma8.4)
==============
Total: 20 (Critical:0 High:13 Medium:6 Low:1 ?:0)
20/20 Fixed, 0 poc, 0 exploits, en: 0, ja: 0 alerts
371 installed

+----------------+------+--------+-----+------+---------+-------------------------------------------------+
|     CVE-ID     | CVSS | ATTACK | POC | CERT |  FIXED  |                       NVD                       |
+----------------+------+--------+-----+------+---------+-------------------------------------------------+
| CVE-2020-26541 |  8.9 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-26541 |
| CVE-2020-8616  |  8.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8616  |
| CVE-2020-8617  |  8.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8617  |
| CVE-2020-8625  |  8.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8625  |
| CVE-2021-25215 |  8.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-25215 |
| CVE-2021-33034 |  8.9 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-33034 |
| CVE-2021-3517  |  8.6 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3517  |
| CVE-2021-3518  |  8.6 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3518  |
| CVE-2021-3520  |  8.6 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3520  |
| CVE-2021-3516  |  7.8 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3516  |
| CVE-2019-6477  |  7.5 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2019-6477  |
| CVE-2020-8623  |  7.5 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8623  |
| CVE-2021-3537  |  7.5 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3537  |
| CVE-2020-8619  |  6.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8619  |
| CVE-2020-8622  |  6.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8622  |
| CVE-2020-8624  |  6.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2020-8624  |
| CVE-2021-20271 |  6.9 |  AV:L  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-20271 |
| CVE-2021-3541  |  6.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2021-3541  |
| CVE-2018-5745  |  4.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2018-5745  |
| CVE-2019-6465  |  3.9 |  AV:N  |     |      |   fixed | https://nvd.nist.gov/vuln/detail/CVE-2019-6465  |
+----------------+------+--------+-----+------+---------+-------------------------------------------------+

```

File: scanner/alma.go (hunk 1 above)
Comment (kazuminn) [reply scanner/alma.go]: I fixed

**Review by MaineK00n — CHANGES_REQUESTED** [review]
Am I correct in assuming that the reason you are changing the processing location of `/etc/centos-release` is because AlmaLinux has `/etc/centos-release`?

- AlmaLinux 8
```console
[root@1539cb6b9627 ~]# ls /etc/*-release
/etc/almalinux-release	/etc/os-release      /etc/system-release
/etc/centos-release	/etc/redhat-release

[root@1539cb6b9627 ~]# cat /etc/centos-release 
AlmaLinux release 8.4 (Electric Cheetah)
[root@1539cb6b9627 ~]# cat /etc/almalinux-release
AlmaLinux release 8.4 (Electric Cheetah)
[root@1539cb6b9627 ~]# cat /etc/redhat-release 
AlmaLinux release 8.4 (Electric Cheetah)
```

- Rocky Linux 8
```console
[root@39f5a7f41777 ~]# ls /etc/*-release
/etc/os-release  /etc/redhat-release  /etc/rocky-release  /etc/system-release
```

- CentOS 8
```console
[root@b95b419c3937 ~]# ls /etc/*-release
/etc/centos-release  /etc/os-release  /etc/redhat-release  /etc/system-release
```

`/etc/rocky-release` process has been done in two places.
https://github.com/kazuminn/vuls/blob/892ad5c506493b50f7a0eb50da4a92cd59dcd641/scanner/redhatbase.go#L44-L84

Isn't it possible to determine Alma Linux in `/etc/redhat-release` as well?
https://github.com/kazuminn/vuls/blob/892ad5c506493b50f7a0eb50da4a92cd59dcd641/scanner/redhatbase.go#L134-L164

Also, I would be happy if you could fix this as well.
https://github.com/kazuminn/vuls/blob/892ad5c506493b50f7a0eb50da4a92cd59dcd641/scanner/oracle.go#L14-L15

**Comment by kazuminn** [comment]
@MaineK00n 


> Am I correct in assuming that the reason you are changing the processing location of /etc/centos-release is because AlmaLinux has /etc/centos-release?



It worked without changing the location, so I restored it.


> /etc/rocky-release process has been done in two places.

I fixed.

> Isn't it possible to determine Alma Linux in /etc/redhat-release as well?

I fixed.

> Also, I would be happy if you could fix this as well.

everyone happy.




Please. Check code. 🙏

**Review by MaineK00n — CHANGES_REQUESTED** [review]
Did you forget to add it here?
https://github.com/kazuminn/vuls/blob/2a685d0f95c48030544d91f9c2b1171aca83de72/scanner/serverapi.go#L213-L228

I'm nervous, but I'd like the order of Rocky and Alma to be unique. In other words, Rocky → Alma, or Alma → Rocky.
For example, the following is in the order Rocky → Alma.
https://github.com/kazuminn/vuls/blob/2a685d0f95c48030544d91f9c2b1171aca83de72/constant/constant.go#L20-L24

However, in the following, the order is reversed.
The examples are listed here. These are not all of them, so please search carefully.
- https://github.com/kazuminn/vuls/blob/2a685d0f95c48030544d91f9c2b1171aca83de72/oval/redhat.go#L17
- https://github.com/kazuminn/vuls/blob/2a685d0f95c48030544d91f9c2b1171aca83de72/oval/util.go#L439-L440

Please add support for AlmaLinux to vulsdoc as well.
https://github.com/vulsdoc/vuls

File: scanner/redhatbase.go [hunk 2]
```diff
@@ -85,6 +89,27 @@ func detectRedhat(c config.ServerInfo) (bool, osTypeInterface) {
 		}
 	}
 
+	if r := exec(c, "ls /etc/almalinux-release", noSudo); r.isSuccess() {
+		if r := exec(c, "cat /etc/alma-release", noSudo); r.isSuccess() {
```
Comment (MaineK00n) [inline scanner/redhatbase.go:93]: Isn't `cat /etc/alma-release` a typo for `cat /etc/almalinux-release`?

**Review by MaineK00n — CHANGES_REQUESTED** [review]

File: scanner/redhatbase.go (hunk 2 above)
Comment (kazuminn) [reply scanner/redhatbase.go]: thankyou! 
I fixed.

**Comment by kazuminn** [comment]
> Did you forget to add it here?

I added.

> I'm nervous, but I'd like the order of Rocky and Alma to be unique. In other words, Rocky → Alma, or Alma → Rocky.
For example, the following is in the order Rocky → Alma.
https://github.com/kazuminn/vuls/blob/2a685d0f95c48030544d91f9c2b1171aca83de72/constant/constant.go#L20-L24

unified to Alma -> Rocky.

**Review by MaineK00n — APPROVED** [review]
LGTM

**Pull request merged**